srcpkgs/pgbackrest/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'pgbackrest'
+pkgname=pgbackrest
+version=2.15.1
+revision=1
+archs="i686 x86_64* aarch64*"
+wrksrc="${pkgname}-release-${version}"
+build_wrksrc="src"
+build_style=gnu-configure
+hostmakedepends="perl"
+makedepends="libssl47 libressl-devel libxml2-devel perl zlib-devel"
```

Comment:
remove `libssl47` and `perl` from makedepends

Reply:
Perl is required for `makedepends` because it errors out else:
> configure: error: library 'perl' is required

`libssl47` was left from earlier tries, fixed now.

Reply:
then it should be `perl-devel`, or if it needs to run `perl` it should be in `hostmakedepends`